docs/upload_mapping_file.md

```diff
@@ -1,71 +1,91 @@
 ## Overview
 
-Error Tracking processes errors collected from the Android SDK. To quickly get started with error tracking download the latest version of [dd-sdk-android][1].
+Error Tracking processes errors collected from the RUM Android SDK. 
 
-If your mobile Android source code is obfuscated, upload your Proguard/R8 mapping file to Datadog so that your different stack traces can be deobfuscated. For a given error, you can get access to the file path, the line number, as well as a code snippet for each frame of the related stack trace.
+Enable Android Crash Reporting and Error Tracking to get comprehensive crash reports and error trends with Real User Monitoring. With this feature, you can access:
+
+- Aggregated Android crash dashboards and attributes
+- Deobfuscated Android crash reports
+- Trend analysis with Android error tracking
+
+Your crash reports appear in [**Error Tracking**][1].
+
+## Setup
+
+If you have not set up the Android SDK yet, follow the [in-app setup instructions][2] or see the [Android RUM setup documentation][3].
+
+1. Download the latest version of the [RUM Android SDK][4].
+2. Configure your application's `version`, `env`, and `service` when [initializing the SDK][5].
```

Comment:
The `version` and `service` are captured automatically, so customers don't have to set it manually, but they need to set their `variant`.
```suggestion
2. Configure your application's `env`, and `variant` when [initializing the SDK][5].
```